### DISCUSSION TITLE
PSDebug trace of `Set-PsFzfOption`

### DISCUSSION THREAD

**Discussion by zmoon** [kelleyma49/PSFzf#159] [Q&A] [answered]
Just curious what this is about (i.e., why so many repeated similar lines?). This is from `Set-PSDebug -Trace 1` with `Set-PsFzfOption -PSReadlineChordReverseHistory 'Ctrl+r'` in my profile. Those repeated similar lines continue for a few more pages.

![image](https://user-images.githubusercontent.com/15079414/174224820-6fb871ea-718f-4de1-9f59-a95815c268bd.png)

**Comment by kelleyma49** [comment, marked as answer]
The repeated lines are based on the number of results from `Get-PSReadLineKeyHandler`. For example, I have 73 shortcuts in my configuration: 

```PowerShell
> Get-PSReadLineKeyHandler | Measure-Object

Count             : 73
Average           :
Sum               :
Maximum           :
Minimum           :
StandardDeviation :
Property          :
```

**Reply by zmoon** [reply]
Got it, thanks for the explanation!
